sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -1282,6 +1282,50 @@ object GpuOverrides {
         override def convertToGpu(child: Expression): GpuExpression = GpuLower(child)
       })
       .incompat(CASE_MODIFICATION_INCOMPAT),
+    expr[StringLPad](
+      "Pad a string on the left",
+      (in, conf, p, r) => new TernaryExprMeta[StringLPad](in, conf, p, r) {
+        override def tagExprForGpu(): Unit = {
+          if (!isLit(in.len)) {
+            willNotWorkOnGpu("only literal length is supported")
+          }
+
+          val padLit = extractLit(in.pad)
+          if (padLit.isEmpty) {
+            willNotWorkOnGpu("only literal pad is supported")
+          } else if (padLit.get.value != null &&
+            padLit.get.value.asInstanceOf[UTF8String].toString.length != 1) {
+            willNotWorkOnGpu("only a single character is supported for pad")
+          }
+        }
+        override def convertToGpu(
+                                     str: Expression,
```

Comment:
Nit: indentation is off here.

Reply:
IntelliJ is setting that to the indentation with the settings we have published for it.  Don't know if it is a regression in IntelliJ or what, but I'll fix it for these

Reply:
I saw these were updated, but the style guide doesn't call for aligning arguments with the open parenthesis.  Instead it should be a continuation (i.e.: 4-space indent from the start of the previous line).  Apparently the style settings that are published are messed up, I'll look into it and fix if necessary.

Reply:
I checked the [IntelliJ settings file](https://github.com/NVIDIA/spark-rapids/blob/branch-0.2/docs/dev/idea-code-style-settings.xml#L32-L37) and it appears to be correct.  Specifically it calls out these settings which should not have aligned the parameters on the opening parenthesis:
```
  <codeStyleSettings language="Scala">
    <option name="ALIGN_MULTILINE_PARAMETERS" value="false" />
    <indentOptions>
      <option name="CONTINUATION_INDENT_SIZE" value="4" />
    </indentOptions>
  </codeStyleSettings>
```